src/blocks/review-comparison/edit.js

```diff
@@ -0,0 +1,240 @@
+/** @jsx jsx */
+
+/**
+ * External dependencies
+ */
+import {
+	css,
+	jsx
+} from '@emotion/react';
+
+import classnames from 'classnames';
+
+import getSymbolFromCurrency from 'currency-symbol-map';
+
+/**
+  * WordPress dependencies.
+  */
+import { __ } from '@wordpress/i18n';
+
+import apiFetch from '@wordpress/api-fetch';
+
+import {
+	Fragment,
+	useEffect,
+	useState
+} from '@wordpress/element';
+
+/**
+  * Internal dependencies
+  */
+import defaultAttributes from './attributes.js';
+import Placeholder from './placeholder.js';
+import Controls from './controls.js';
+import Inspector from './inspector.js';
+import { blockInit } from '../../helpers/block-utility.js';
+import defaultReviewAttributes from '../review/attributes.js';
+import {
+	StarFilled,
+	StarHalf
+} from '../../helpers/icons.js';
+
+let tableImages = [];
+let tableName = [];
+let tablePrice = [];
+let tableRating = [];
+let tableDescription = [];
+let tableStatistics = [];
+let tableLinks = [];
```

Comment:
Those should not be some states as well or a single object that contains them all?

Reply:
Not state because state is discarded when component refreshes, variable it outside the component so stays intact.